autopilot/contractor.go

```diff
@@ -440,18 +531,34 @@ func (c *contractor) renewFundingEstimate(cfg Config, id types.FileContractID) (
 	// fetch contract
 	contract, err := c.ap.bus.Contract(id)
 	if err != nil {
+		c.logger.Errorw(
+			"missing contract",
+			"hk", contract.HostKey,
+			"fcid", contract.ID,
+			"err", err,
```

Comment:
No, the field is `msg` by default. This is `Errorf` vs `Errorw`:
`{"level":"error","date":"2022-12-07T11:33:57+01:00","msg":"something failed, err: oh no"}`
`{"level":"error","date":"2022-12-07T11:33:57+01:00","msg":"missing contract", "err":"oh no"}`

Do you prefer having `fmt.Sprintf("missing contract, err: %v", err)` as the message?
I don't feel very strongly but I think I prefer the way I ended up doing it.

Reply:
The idea is to use `fmt.Errorf` to add some context where necessary, `logger.Errorf` to log unexpected errors usually returned by the `Bus` (for instance of `RecordHostInteraction` would error out), and use `logger.Errorw` or `logger.Debugw` in all other cases where we want to provide more context around the error/event.

Reply:
Adding context is fine it just seems a bit weird that sometimes I find the error message in `err` and sometimes in `msg`.  Which could make it a bit harder to parse the file by hand and look for substrings in the error messages. Because I wouldn't know whether it ended up in `err` or `msg`.